src/content/development/contribution-guide/_index.en.md

```diff
@@ -234,6 +235,18 @@ During the week when the general availability release is performed, the next ver
 Additionally, a retrospective meeting for the last release cycle will be held.
 There will be no dedicated [test day](#test-days), as the release candidate has been tested and no changes have occurred since.
 
+##### Release Notes
+
+[Release notes] should be published for each GA version.
+All new features and bug fixes of the release should be labeled with `release-note-needed` and have a relevant release note
+documented as part of the PR. Release notes submitted with PRs should be written in individual files in the `/release-notes` directory
+of the relevant repository. The release note file name should be prefixed with a date in the format `YYYYMMDD` and end with a `.md` suffix.
+The content of the file should be a Markdown snippets.
+Once a week, as part of the tasks and bugs triage meeting, the team will make sure that new bugs / tasks / PR's are properly
+labeled and have the release notes documentation. When the version is ready for GA, all the notes can then be gathered
+from issues / PR's using the `release-note-needed` label and a [pull request to the website repository] should be opened.
```

Comment:
```suggestion
from issues / PRs using the `release-note-needed` label and a [pull request to the website repository] should be opened.
```

Also, so far I think we've been following the pattern with backports and looking at the presence of `release-note-needed` and the absence of `release-note-handled`.

We may want to actually link to the relevant query:

https://github.com/pulls?q=is%3Apr+user%3Asubmariner-io+label%3Arelease-note-needed+-label%3Arelease-note-handled+

Reply:
That's a good point, I'll add a query to include both PRs and issues:
https://github.com/pulls?q=user%3Asubmariner-io+label%3Arelease-note-needed+-label%3Arelease-note-handled+